package.json

```
@@ -15,7 +15,7 @@
     "url": "https://github.com/SonarSource/sonar-scanner-npm/issues"
   },
   "license": "LGPL-3.0-only",
-  "main": "src/index.js",
+  "main": "build/src/index.js",
```

Once we generate type declaration we can specify them here, so that TypeScript projets can import this package and have types out of the box

```suggestion
  "main": "build/src/index.js",
  "types": "build/src/index.d.ts",
```